src/ucstring.rs

```diff
@@ -770,9 +789,8 @@ impl UCString<u32> {
     /// assert!(res.is_err());
     /// assert_eq!(res.err().unwrap().nul_position(), 1);
     /// ```
-    pub fn from_chars(v: impl Into<Vec<char>>) -> Result<Self, NulError<u32>> {
```

Comment:
Why change the signature here? this makes this PR a breaking change. (same across multiple changes in this file)